content/docs/overview.md

```diff
@@ -1233,6 +1233,9 @@ case c..=d:
 
 bit_set[a..<b]
 bit_set[a..=b]
+
+foo := [?]int{0..=3 = 1} // inits array: [1, 1, 1, 1]
```

Comment:
I don't like "inits array:" here, "Initialises as:" seems better to me, also open to another better equivalent.

Reply:
sorry for the delay - fixed